autoscaling/custom-compute-classes/reservations/ccc-any-reservation.yaml

```diff
@@ -0,0 +1,30 @@
+# Copyright 2024 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# [START gke_autoscaling_ccc_any_reservation]
+apiVersion: cloud.google.com/v1
+kind: ComputeClass
+metadata:
+  name: ccc-reservations
+spec:
+  nodePoolAutoCreation:
+    enabled: true
+  priorities:
+    - machineType: ${MACHINE_TYPE}
```

Comment:
**Issue:** Thoughts on using in-docs (editable within the cloud.google.com page) variables for some of these variables? See [this pull-request for more info](https://github.com/GoogleCloudPlatform/kubernetes-engine-samples/pull/1531/files). In this case, you'll want to remove the `${` and `}`.